fe/src/main/java/org/apache/doris/qe/ConnectProcessor.java

```diff
@@ -92,12 +93,16 @@ private void handlePing() {
         ctx.getState().setOk();
     }
 
-    private void auditAfterExec(String origStmt, StatementBase parsedStmt) {
+    private void auditAfterExec(String origStmt, StatementBase parsedStmt,
+                StmtExecutor.QueryStatistic queryStatistic) {
         // slow query
         long elapseMs = System.currentTimeMillis() - ctx.getStartTime();
         // query state log
         ctx.getAuditBuilder().put("state", ctx.getState());
         ctx.getAuditBuilder().put("time", elapseMs);
+        Preconditions.checkNotNull(queryStatistic); 
+        ctx.getAuditBuilder().put("cpu", queryStatistic.getFormattingCpu());
+        ctx.getAuditBuilder().put("io", queryStatistic.getFormattingIo());
```

Comment:
use scanRows, processRows instead of cpu and io. because a man can't know what's io/cpu stand for

Reply:
ok, i will fix it.